src/screens/OtherProfile/index.js

```diff
@@ -205,14 +205,22 @@ const OtherProfile = () => {
     const __renderBlockIcon = () => {
       if (isBlocking)
         return (
-          <View style={styles.buttonFollowing}>
-            <Text style={styles.textButtonFollowing}>Blocked</Text>
+          <View style={styles.buttonFollowing(isAnonimity)}>
+            <Text style={styles.textButtonFollowing(isAnonimity)}>Blocked</Text>
           </View>
         );
 
       return (
-        <View style={{...styles.btnMsg, borderColor: colors.gray1}}>
-          <BlockIcon width={20} height={20} style={{color: colors.gray1}} />
+        <View
+          style={{
+            ...styles.btnMsg(isAnonimity),
+            borderColor: isAnonimity ? COLORS.anon_primary : COLORS.signed_primary
+          }}>
+          <BlockIcon
+            width={20}
+            height={20}
+            style={{color: isAnonimity ? COLORS.anon_primary : COLORS.signed_primary}}
+          />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in this section are primarily related to the color scheme. The color values have been updated to use the new `COLORS` object from the `theme` module, which is a good practice for maintaining consistency across the application. However, it's important to ensure that the new colors (`COLORS.anon_primary`, `COLORS.signed_primary`) are defined in the `COLORS` object and they match the intended design.



<!-- This is an auto-generated comment by OSS CodeRabbit -->